rustv1/examples/comprehend/src/getting_started/tests/mod.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use aws_config::BehaviorVersion;
+use aws_sdk_comprehend::Client;
+use crate::getting_started::scenario::ComprehendScenario;
+
+#[tokio::test]
+async fn test_comprehend_scenario() {
+    let config = aws_config::defaults(BehaviorVersion::latest())
+        .load()
+        .await;
+    let client = Client::new(&config);
+    let scenario = ComprehendScenario::new(client);
+
+    // Run the scenario - this will make real API calls
+    let result = scenario.run().await;
+    
+    // The scenario should complete successfully
+    assert!(result.is_ok(), "Comprehend scenario should complete successfully: {:?}", result);
+}
+
+#[tokio::test]
```

Comment:
These are full integration tests, I'm not sure if I support this. Since they require network access.